vsphere/datadog_checks/vsphere/api.py

```diff
@@ -146,9 +146,9 @@ def smart_connect(self):
         self.log.debug("Connected to %s", version_info.fullName)
 
     @smart_retry
-    def check_health(self):
```

Comment:
NOTE: `check_health` method was initially used for service check, but it's not used anymore.